pkg/ebpf/tracee.go

```diff
@@ -779,15 +779,6 @@ func (t *Tracee) initDerivationTable() error {
 	return nil
 }
 
-// RegisterEventDerivation registers an event derivation handler for tracee to use in the event pipeline
-func (t *Tracee) RegisterEventDerivation(deriveFrom events.ID, deriveTo events.ID, deriveCondition func() bool, deriveLogic derive.DeriveFunction) error {
-	if t.eventDerivations == nil {
-		return errfmt.Errorf("tracee not initialized yet")
-	}
-
-	return t.eventDerivations.Register(deriveFrom, deriveTo, deriveCondition, deriveLogic)
```

Comment:
Good catch, @rscampos.

@NDStrahilevitz, I noticed they were introduced by you. Is there a plan for its use?

Reply:
exactly, was introduced at https://github.com/aquasecurity/tracee/pull/2551/, but `RegisterEventDerivation` was never used.

Reply:
This was meant to be used in plugins/extensions whenever they are introduced, so that one could add a new derived event. Seeing as we intend to merge derived events and signatures eventually we will need such an exported method.
However, we are a while away from this goal, so i'm ok with deleting this method as we may need to refit it anyway for whatever API we come up with.

Reply:
So I'll rebase it removing both.